src/api/app/assets/javascripts/webui/request.js

```diff
@@ -79,25 +79,17 @@ function prefillSubmitRequestForm() {
   });
 }
 
-function requestAddReviewAutocomplete() { // jshint ignore:line
+function requestAddAutocomplete(element) { // jshint ignore:line
   $('.modal').on('shown.bs.modal', function() {
     $('.hideable input:not(:visible)').attr('disabled', true);
   });
 
-  $('#review_type').change(function () {
+  $(element).change(function () {
```

Comment:
Element is the most general base class from which all element objects (i.e. objects that represent elements) in a Document inherit.  -> https://developer.mozilla.org/en-US/docs/Web/API/Element